src/celeritas/em/UrbanMscParams.cc

```diff
@@ -137,6 +139,16 @@ UrbanMscParams::UrbanMscParams(ParticleParams const& particles,
         return &imm->xs_table;
     };
 
+    {
+        // Set initial high/low energy limits
+        auto const& phys_vec
+            = get_scaled_xs(host_data.ids.electron)->physics_vectors;
+        CELER_ASSERT(!phys_vec.empty());
+        CELER_ASSERT(!phys_vec[0].x.empty());
+        host_data.params.low_energy_limit = MevEnergy{phys_vec[0].x.front()};
```

Comment:
We might want to double check that the MSC lambda values aren't constant outside the grid in Geant4. It looks like MSC is applicable in Geant4 if the energy is above `lowestKinEnergy` in `G4VMultipleScattering`, which is 10 eV.

Reply:
I assume since it uses physics vectors, the cross sections are extrapolated out as being constant. For our immediate purposes, the electron tracking cut should make this change irrelevant. For low-energy physics, we might want to check with an expert about the models and various cutoffs they choose.

Reply:
Yeah it probably make much difference.. but since MSC happens before energy loss in along-step we can still have particles with energies below the hard cut at that point.

Reply:
That seems to me like an erroneous inconsistency between the process and the models. The process shouldn't have a tiny gap between 10 eV and 100 eV that implicitly extrapolates cross sections and uses the lowest-energy model?